component/common/loki/net/config.go

```diff
@@ -0,0 +1,111 @@
+// Package http contains a River serializable definition of the weaveworks weaveworks config in
+// https://github.com/weaveworks/common/blob/master/server/server.go#L62.
+package net
+
+import (
+	"flag"
+	"time"
+
+	weaveworks "github.com/weaveworks/common/server"
+)
+
+const (
+	DefaultHTTPPort = 9090
+	DefaultGRPCPort = 9091
```

Comment:
Prometheus listens to 9090 by default. While rare, it _may_ trip up some users?

How do you feel about using 8080/8081 here?

Reply:
Sure thing, updating